### PR TITLE
fix: removing sts call for usage data

### PR DESCRIPTION
**Pull request by ammarkarachi**
#### Description of changes
Removed sts call for usage-data 

#### Checklist

- [x] PR description included
- [x] `yarn test` passes
- [x] Tests are [changed or added](https://github.com/aws-amplify/amplify-cli/blob/master/CONTRIBUTING.md#tests)
- [ ] Relevant documentation is changed or added (and PR referenced)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by jhockett — APPROVED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -191,8 +191,6 @@ jobs:
             cd packages/graphql-transformers-e2e-tests/ && yarn e2e
             --maxWorkers=3
           no_output_timeout: 90m
-          environment:
-            AMPLIFY_CLI_DISABLE_LOGGING: "true"
```
Comment (kaustavghosh06) [inline .circleci/config.yml:195]: Is this removal intentional?

Comment (ammarkarachi) [reply .circleci/config.yml:195]: No I think it got auto generated

Comment (kaustavghosh06) [reply .circleci/config.yml:195]: Please revert this back. This was added recently as a part of this PR - https://github.com/aws-amplify/amplify-cli/pull/7087/files

Comment (SwaySway) [reply .circleci/config.yml:195]: Found out this needs to get added into the base config as well
https://github.com/aws-amplify/amplify-cli/pull/7093

Comment (ammarkarachi) [reply .circleci/config.yml:195]: cc: @SwaySway

File: packages/amplify-cli/src/context-manager.ts [hunk 2]
```diff
@@ -19,13 +19,12 @@ export async function attachUsageData(context: Context) {
...
     ? AMPLIFY_CLI_ENABLE_USAGE_DATA === 'true'
     : config.usageDataConfig.isUsageTrackingEnabled;
-  const accountId = await context.amplify.executeProviderUtils(context, 'awscloudformation', 'getAccountId');
   if (usageTrackingEnabled) {
     context.usageData = UsageData.Instance;
   } else {
     context.usageData = NoUsageData.Instance;
   }
-  context.usageData.init(config.usageDataConfig.installationUuid, getVersion(context), context.input, accountId, getProjectSettings());
+  context.usageData.init(config.usageDataConfig.installationUuid, getVersion(context), context.input, '', getProjectSettings());
```
Comment (SwaySway) [inline packages/amplify-cli/src/context-manager.ts:27]: Will we need that empty string argument later?

Comment (ammarkarachi) [reply packages/amplify-cli/src/context-manager.ts:27]: I will need it or it will break validations on the back end

**Review by SwaySway — APPROVED** [review]
